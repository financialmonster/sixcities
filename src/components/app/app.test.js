import React from 'react';
import renderer from 'react-test-renderer';
import App from './app.jsx';

it(`Correctly renders`, () => {
    const tree = renderer.create(<App onClick={jest.fn()}/>).toJSON();

    expect(tree).toMatchSnapshot();
});
