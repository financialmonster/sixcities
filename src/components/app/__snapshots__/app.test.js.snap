// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing of markup Correctly renders 1`] = `
<p
  onClick={[MockFunction]}
>
  Hello!!
</p>
`;
